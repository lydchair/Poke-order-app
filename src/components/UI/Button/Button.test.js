import React from 'react';
import renderer from 'react-test-renderer';
import Button from './Button';



test('<Button /> renders with text as prop', () => {
  const tree = renderer.create(
    <Button />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

