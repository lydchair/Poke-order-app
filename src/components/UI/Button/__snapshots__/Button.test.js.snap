// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders with text as prop 1`] = `
<button
  className=" "
  disabled={undefined}
  onClick={undefined}
/>
`;
